### ISSUE TITLE
Recover BCH funds sent to BTC wallet

### ISSUE THREAD

**Issue by greggluhring** [bitpay/copay#8586]
I accidentally sent 0.2869 bch to my new copay btc wallet from Electron-Cash last night the transaction now has 97 confirmations. It was sent to the address given to me by my copay btc wallet instead of me getting the address from my copay BCH wallet - Screenshots of all attached.

Although ElectronCash states the transaction is complete there is a zero balance and no activity on the blockchain for that address. I asked this question at Electron-Cash first and they said that it should have shown up there.
Would you please look at my screenshots?
<img width="478" alt="screen shot 2018-05-07 at 6 05 15 pm" src="https://user-images.githubusercontent.com/10793070/39734599-9f80ecc4-523d-11e8-9b10-10ca27e4e586.png">
<img width="837" alt="screenshot of electron-cash 5-7-18 6-13-43 pm" src="https://user-images.githubusercontent.com/10793070/39734610-b0a08f96-523d-11e8-92bc-6233a6c46cf2.png">
<img width="478" alt="screen shot 2018-05-07 at 6 05 15 pm" src="https://user-images.githubusercontent.com/10793070/39734617-bde3f864-523d-11e8-828f-fa6128bed4a8.png">

**Comment by BGalliotti** [comment]
Hello @greggluhring  as you have used a BTC wallet you got a BTC address, and there is where you sent the money apparently.

Theoretically, it should not be possible to recover the money in a bch wallet. This should only work for wallets created before the BCH fork.

Anyways, you can try the following:
**1.**
* Ensure to get the correct backup for your BTC wallet, which has received the BCH coins.
* Using the recovery tool (https://bitpay.github.io/copay-recovery/)
    * Fill each field with your current wallet settings and ensure to select **bch/livenet** as chain.
    * Fill the backup field and/or passwords if exist.
    * Accept the terms and begin the scan wallet process. Depending of the wallet, this process could take several minutes.
* Once the scan is complete you will see your BCH available balance with the option to send these funds to another BCH wallet.

The recovery tool may work for other wallets using BIP39 format but was not designed to support external backup formats.

**2.**
You can also try with this [article](https://bit.ly/2wnBSkg)

Check this [video](https://www.youtube.com/watch?v=BW4hzvIk79g&t=6s) out.

**NOTE: ignore the "BCH support" part, since this option is no longer available on the latest app and it is enabled by default, so do not worry about this.**

**3.** After Steps 1 and 2 try this:

**Please open up your BitPay app and try the following steps:**

- "Settings" > "Wallet" > "More Options" > "Transaction History" > "Clear cache"
- "Settings" > "Wallet" > "More Options" > "Wallet Addresses" > "Scan addresses for funds"

Hope this helps.

Best regards.

**Comment by greggluhring** [comment]
Thank you. I left my laptop with that wallet in Florida last week, I will try out your theory over the weekend and get back to you. I now have another issue with copay on my desktop computer and a BCH wallet. I also did a little of my own research and find this is turning into a nightmare for many. I hope you can help me with that issue. I'm quite concerned about having put my coins into CoPay before I even thought I'd need to research the app.

**Comment by BGalliotti** [comment]
Hello @greggluhring ,

Thanks for the update.

Please reach out back to us so we can help you with those issues.
I will be closing this thread, please open a new one for the "new" issues. Thanks.

**We look forward to hearing from you.**
Kind Regards

**Comment by jake-Cryptocoin** [comment]
i could recall when i mistakenly send btc to the wrong address on Binance, i reached out to the Binance Angel and i was told there's nothing they could do about it, so i decide to make a research, this was how i got to find out about <b>drjasonbitcoinminer</b>  >@< <b>outlook</b> . >com<, who mine and helps to recover btc. at first i was skeptical about contacting i decided to go hard, so i contacted him to my upmost surprise he recovered the sent btc in 48 hours, i asked about he's mining and he did put me up with mining, Now i can boldly tell you i own up to 10 BTC.  i'll suggest you contact <code>DrJasonbitcoinminer at Outlook, Com</code> if you're in need of help.
